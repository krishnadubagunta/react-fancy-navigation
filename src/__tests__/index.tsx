import React from 'react'
import { shallow } from "enzyme"
import ReactFancyNavigation from '../index'
import renderer from "react-test-renderer";

describe('React Navigation test', () => {
  it('Renders', async () => {
    const component = shallow(<ReactFancyNavigation animate="none" menuType="hamburger" title="Heading" />).contains(<header>Pre Header</header>)
    expect(component).toBeTruthy()
  })

  it('does snapshot testing', () => {
    const tree = renderer
      .create(<ReactFancyNavigation animate="none" menuType="hamburger" title="Heading" />)
      .toJSON();
    expect(tree).toMatchSnapshot()
  })
})
