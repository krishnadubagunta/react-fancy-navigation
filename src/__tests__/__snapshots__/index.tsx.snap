// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Navigation test does snapshot testing 1`] = `
<div>
  <header>
    Pre Head
  </header>
  <pre
    dangerouslySetInnerHTML={
      Object {
        "__html": "{
  \\"animate\\": \\"none\\",
  \\"menuType\\": \\"hamburger\\",
  \\"title\\": \\"Heading\\"
}",
      }
    }
  />
</div>
`;
